Files/App.xaml.cs

```diff
@@ -359,6 +362,18 @@ protected override async void OnActivated(IActivatedEventArgs args)
                         }
                     }
                     break;
+                /* For future use
+                    case ActivationKind.ToastNotification:
+                    var eventArgsForNotification = args as ToastNotificationActivatedEventArgs;
+                    rootFrame.Navigate(typeof(MainPage), "tryRecoverTabs", new SuppressNavigationTransitionInfo());
+                    // Ensure the current window is active.
+                    Window.Current.Activate();
+                    Window.Current.CoreWindow.PointerPressed += CoreWindow_PointerPressed;
+                    Window.Current.CoreWindow.Activated += CoreWindow_Activated;
+                    currentView.BackRequested += Window_BackRequested;
+                    // Launch the URI now
+                    SettingsViewModel.ReportIssueOnGitHub();
+                    return;*/
```

Comment:
We shouldn't add code that is not being used, it would be better to add this later when it's ready to be used.